CMakeLists.txt

```diff
@@ -103,6 +114,13 @@ if (WIN32)
         contrib/imgui/backends/imgui_impl_glfw.cpp
         ${CMAKE_CURRENT_SOURCE_DIR}/src/platform/windows/app.manifest
     )
+
+    if (MSVC)
+        # Disable RTTI
+        # Enable Multi Processor Compilation
+        target_compile_options(SonyHeadphonesClient PRIVATE /GR- /MP)
```

Comment:
With `/MP` - [CMake's jobs (`-j`) option](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-build-j) should be preferred over this - with build systems like `ninja` this would be irrelevant too since they'd schedule sources one-by-one themselves, in parallel.

And with turning RTTI off - add this to other compilers too!

Reply:
I don't have that much experience with CMake, as well as experience with developing native code for platforms other than Windows, sorry. Might need your help for this one.